doc/defx.txt

```diff
@@ -706,6 +706,13 @@ OPTIONS							*defx-options*
 
 		Default: 90
 
+						*defx-option-winborder*
```

Comment:
Please adjust the indent to the above option.

Reply:
I tried to find out what happened here since it aligned in my editor. https://github.com/Shougo/defx.nvim/blob/0381ae75beca238c3a787adde0e59f21926199c4/doc/defx.txt#L709 if you look at the file individually it also seems to align fine. I think it just looks odd because github suddenly renders tabs as 4 characters wide instead of 8 in a diff, but I'm not sure.

If I'm wrong please let me know what I should do to fix instead.